### PR TITLE
Fix/issue 11669 block icon

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Fixes https://github.com/WordPress/gutenberg/issues/11669

## How has this been tested?

- Linting
- Browser testing
- Local e2e
- Local Unit

## Types of changes
Converted disabled block icon buttons to a span so can can be semantically presentational and removed from assistive technology.

## Checklist:
- [ x ] My code is tested.
- [ x ] My code follows the WordPress code style. 
- [ x ] My code follows the accessibility standards. 
- [ x ] My code has proper inline documentation.

### CODE REVIEW

File: packages/editor/src/components/block-switcher/index.js [hunk 1]
```diff
@@ -61,15 +61,11 @@ export class BlockSwitcher extends Component {
 				return null;
 			}
 			return (
-				<Toolbar>
-					<IconButton
-						disabled
-						className="editor-block-switcher__no-switcher-icon"
-						label={ __( 'Block icon' ) }
-					>
+				<div className="components-toolbar" aria-hidden="true">
```
Comment (jorgefilipecosta) [inline packages/editor/src/components/block-switcher/index.js]: We are using the class of a component outside of it in another element. I guess in this case we should use a different class and add the minimum necessary styles to have a similar UI.

File: packages/editor/src/components/block-switcher/index.js [hunk 2]
```diff
@@ -61,15 +61,11 @@ export class BlockSwitcher extends Component {
...
+					<span className="components-icon-button editor-block-switcher__no-switcher-icon is-presentational">
```
Comment (jorgefilipecosta) [inline packages/editor/src/components/block-switcher/index.js]: Same as in the toolbar case we should avoid the usage of components-icon-button class as it is internal to another component in another package.

**Review by jorgefilipecosta — COMMENTED** [review]
Hi @timwright12, thank you for your contribution I left some suggestions regarding the code.

From the functional and design point of view I think things look as expected only the opacity was removed:
screenshot 2018-12-05 at 12 57 11

File: packages/editor/src/components/block-switcher/index.js (hunk 1 above)
Comment (tofumatt) [reply packages/editor/src/components/block-switcher/index.js]: As mentioned this should be:
```suggestion
				<div className="components-block-switcher" aria-hidden="true">
```

As mentioned by Jorge you'll need to duplicate some styles from `components-toolbar`, I'd imagine, to get this to work.

**Review by tofumatt — CHANGES_REQUESTED** [review]
The classnames need to change here but the implementation seems solid and makes sense to me, so once we get the classnames sorted out this should be good 👍

File: packages/editor/src/components/block-switcher/index.js (hunk 1 above)
Comment (timwright12) [reply packages/editor/src/components/block-switcher/index.js]: @jorgefilipecosta @tofumatt All set here

File: packages/editor/src/components/block-switcher/index.js (hunk 2 above)
Comment (timwright12) [reply packages/editor/src/components/block-switcher/index.js]: @jorgefilipecosta @tofumatt All set here as well. Let me know if you find anything else. Thanks for the feedback!

**Comment by jorgefilipecosta** [comment]
> @jorgefilipecosta hi, was going through the issues with the "Needs accessibility feedback" label: what kind of feedback is needed here?

Hi @afercia, I guess we just need to compare with the previous version and check if the main objective was accomplished: The disable block icon is semantically presentational and is removed from assistive technologies.

File: packages/editor/src/components/block-switcher/index.js [hunk 3]
```diff
@@ -61,15 +61,11 @@ export class BlockSwitcher extends Component {
 				return null;
 			}
 			return (
-				<Toolbar>
-					<IconButton
-						disabled
-						className="editor-block-switcher__no-switcher-icon"
-						label={ __( 'Block icon' ) }
-					>
+				<div className="components-block-switcher" aria-hidden="true">
```
Comment (gziolo) [inline packages/editor/src/components/block-switcher/index.js]: I don't think, we should be re-creating `Toolbar` here as this is still the same `div`.

**Review by gziolo — CHANGES_REQUESTED** [review]
I left my comments. I also noticed that this PR needs to be refreshed with the latest changes from the `master` branch.

**Comment by afercia** [comment]
From an accessibility perspective, looks good to me.

**Comment by mapk** [comment]
Looks like this has been fixed with https://github.com/WordPress/gutenberg/pull/13721 so I'm closing this PR. If I missed anything we can reopen if needed.

**Pull request closed without merging**